xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -339,6 +341,12 @@ bool CSystemGUIInfo::GetLabel(std::string& value, const CFileItem *item, int con
       value = g_langInfo.GetRegionLocale();
       return true;
     }
+
+    case SYSTEM_PVR_COUNT:
+    {
+      value = StringUtils::Format("{}", CServiceBroker::GetPVRManager().Clients()->EnabledClientAmount());
```

Comment:
Instead of `StringUtils::Format` you could use `std::to_string`

Reply:
Done.